client/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "reflextion-client",
+  "version": "0.0.1",
+  "description": "Client-side for reflextion app",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --config webpack.prod.js",
+    "start": "webpack serve --config webpack.dev.js --progress",
+    "lint": "eslint src & prettier src --write",
```

Comment:
i think, command lint just should check lint, but do not fix issues. Can be case, that you don't want to fix smth or it will broke you code. Better first check by linter, then check by yourself and then fix if needed